appdb/id.lproj/Localizable.strings

```diff
@@ -0,0 +1,400 @@
+"File size not found in resource values" = "Ukuran file tidak ditemukan dalam nilai sumber daya";
+"A report has already been submitted" = "Laporan sudah dikirim";
+"Installation failed, but can be fixed from Settings -> Device Status" = "Instalasi gagal, tetapi dapat diperbaiki dari Pengaturan -> Status Perangkat";
+"An error has occurred: malformed JSON" = "Terjadi kesalahan: JSON salah format";
+"Your PRO subscription has expired." = "Langganan PRO Anda telah kedaluwarsa.";
+"Your PRO subscription has been revoked by Apple." = "Langganan PRO Anda telah dicabut oleh Apple.";
+"Your device doesn't seem to have a PRO subcription." = "Perangkat Anda sepertinya tidak memiliki langganan PRO.";
+"App duplication is currently supported on non-jailbroken devices with PRO." = "Duplikasi aplikasi saat ini didukung pada perangkat yang tidak di-jailbreak dengan PRO.";
+"Unknown voucher partner." = "Mitra voucher tidak dikenal.";
+"Invalid voucher." = "Voucher tidak valid.";
+"Voucher already used." = "Voucher sudah digunakan.";
+"No devices with this email." = "Tidak ada perangkat dengan email ini.";
+
+////////////////////////////
+//     LINK REPORTING     //
+////////////////////////////
+
+"Send" = "Kirim";
+"Report" = "Lapor";
+"Reporting a broken link for '%@'." = "Laporkan link rusak untuk '%@'.";
+"Enter a reason for your report" = "Masukan alasan pelaporan";
+"Link reported successfully!" = "Tautan berhasil dilaporkan!";
+
+/////////////////////
+//     CREDITS     //
+/////////////////////
+"Credits" = "Kredit";
+"Developer" = "Developer";
+"Special Thanks" = "Terima kasih";
+"Translations" = "Penerjemah";
+"appdb team" = "tim appdb";
+"Icon and banner designer" = "Desainer ikon dan spanduk";
+"View project on GitHub" = "Lihat proyek di GitHub";
+
+/////////////////////////////////////////
+//     WISHES / AUTOMATIC REQUESTS     //
+/////////////////////////////////////////
+"Wishes" = "Keinginan";
+"New" = "Baru";
+"Processing" = "diproses";
+"Failed" = "Gagal";
+"Fulfilled" = "Terpenuhi";
+"Free" = "Gratis";
+"Enter AppStore URL" = "Masukan URL Appstore";
+"Enter below the AppStore URL of the app you'd like to request" = "Masukkan URL Aplikasi dari AppStore yang ingin Anda minta";
+"Add" = "Tambah";
+"App has been requested successfully!" = "Aplikasi telah berhasil diminta!";
+"Price: %@" = "Harga: %@";
+"Version: %@" = "Versi: %@";
+"No new wishes to show" = "Tidak ada keinginan baru untuk ditampilkan";
+"No fulfilled wishes to show" = "Tidak ada keinginan yang terpenuhi untuk ditampilkan ";
+
+///////////////////
+//     OTHER     //
+///////////////////
+
+"Cancel" = "Batalkan";
+"Dismiss" = "Berhentikan";
+"Yes" = "Ya";
+"OK" = "OK";
+"No" = "Tidak";
+"Done" = "Selesai";
+"more" = "lebih banyak";
+"Open in Safari" = "Buka di Safari";
+"%@s lalu" = "%@s ago"; // seconds, e.g. 15s ago
+"%@m lalu" = "%@m ago"; // minutes, e.g. 5m ago
+"%@h lalu" = "%@h ago"; // hours, e.g. 6h ago
+"%@d lalu" = "%@d ago"; // days, e.g. 3d ago
```

Comment:
I think you got these flipped! Should be:
```
"%@s ago" = "%@s lalu"; // seconds, e.g. 15s ago
"%@m ago" = "%@m lalu"; // minutes, e.g. 5m ago
"%@h ago" = "%@h lalu"; // hours, e.g. 6h ago
"%@d ago" = "%@d lalu"; // days, e.g. 3d ago
```
Will merge as soon as you fix this! Thanks

Reply:
solved